src/index.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Messenger from './pages/Messenger';
-import registerServiceWorker from './registerServiceWorker';
+import Commit from 'pages/Commit';
+
+import registerServiceWorker from 'registerServiceWorker';
 
 import {
-    BrowserRouter as Router,
-    Route,
+  BrowserRouter as Router,
+  Route
 } from 'react-router-dom';
 
+
+import './index.css';
+
 const App = () => (
-  <Router basename="/">
-    <Route exact path="/" component={Messenger} />
+  <Router basename="/commit">
```

Comment:
Why have this be the base?

Reply:
Oh, if we are gonna expose this via github, we need the base path the same as the name of the repo.

Reply:
kk